install.py

```diff
@@ -464,10 +458,18 @@ def validate_path(path):
         cmake_flags += ["-Dlegate_core_LEGION_BRANCH=%s" % legion_branch]
 
     cmake_flags += extra_flags
+    ninja_flags = [f"-j{str(thread_count)}"]
```

Comment:
Can we call this something other than `ninja_flags`? Since it is relevant to both makefiles and ninja.

Reply:
sure, changed it to `build_flags`